js/content/community.js

```diff
@@ -3700,6 +3700,98 @@ let WorkshopBrowseClass = (function(){
     return WorkshopBrowseClass;
 })();
 
+let EditGuidePageClass = (function(){
+
+    function EditGuidePageClass() {
+        this.allowMultipleLanguages();
+        this.addCustomTags();
+        this.rememberTags();
+    }
+
+    function addTag(name) {
+        let tag = `<div><input type="checkbox" name="tags[]" value="${name}" class="inputTagsFilter" checked>${name}</div>`;
+        HTML.beforeBegin("#es_add_tag", tag);
+    }
+
+    EditGuidePageClass.prototype.allowMultipleLanguages = function() {
+        document.getElementsByName("tags[]").forEach(tag => tag.type = "checkbox");
+    };
+
+    EditGuidePageClass.prototype.addCustomTags = function() {
+        let langSection = document.querySelector("#checkboxgroup_1");
+        if (!langSection) { return; }
+
+        Messenger.addMessageListener("addtag", function(name) {
+            addTag(name);
+        });
+        
+        HTML.afterEnd(langSection,
+            `<div class="tag_category_container" id="checkboxgroup_2">
+                <div class="tag_category_desc">${Localization.str.custom_tags}</div>
+                <div><a style="margin-top: 8px;" class="btn_blue_white_innerfade btn_small_thin" id="es_add_tag">
+                    <span>${Localization.str.add_tag}</span>
+                </a></div>
+            </div>`);
+
+        document.querySelector("#es_add_tag").addEventListener("click", function() {
+            ExtensionLayer.runInPageContext(`function() {
+                let Modal = ShowConfirmDialog("${Localization.str.custom_tags}", \`<div class="commentthread_entry_quotebox"><textarea placeholder="${Localization.str.enter_tag}" class="commentthread_textarea" id="es_tag" rows="1"></textarea></div>\`);
```

Comment:
You can add `text-transform: capitalize` to the `<textarea>`

Reply:
I'm confused, why? The reason why I capitalize the first letter, is because steam does this too, no matter what case your tag starts. This way I can let people know early that the first letter will be capitalized.

Reply:
This will automatically capitalize the input.
I meant this to be an addition to the JS capitalizing